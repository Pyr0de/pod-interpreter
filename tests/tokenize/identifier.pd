identifier
identifier1_2_3
abc23
asb2 hello
2abc
