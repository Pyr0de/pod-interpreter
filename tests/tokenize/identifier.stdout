ExitCode: 0
IDENTIFIER identifier <nil> 1
IDENTIFIER identifier1_2_3 <nil> 2
IDENTIFIER abc23 <nil> 3
IDENTIFIER asb2 <nil> 4
IDENTIFIER hello <nil> 4
INT 2 2 5
IDENTIFIER abc <nil> 5
EOF  null
